### PR TITLE
MaxMind ccpa

### PR DESCRIPTION
**Pull request by escopecz**
Docs for https://github.com/mautic/mautic/pull/8647 and https://github.com/mautic/mautic/pull/8769

### CODE REVIEW

File: pages/02.Setup/03.Cron Jobs/docs.en.md [hunk 1]
```diff
@@ -139,6 +139,25 @@ Use ‘--gdpr’ flag to delete data to fulfill GDPR European regulation. This w
 php /path/to/mautic/bin/console mautic:maintenance:cleanup --days-old=365 --dry-run
 ```
 
+### MaxMind CCPA complience
```
Comment (RCheesley) [inline pages/02.Setup/03.Cron Jobs/docs.en.md:142]:
```suggestion
### MaxMind CCPA compliance
```

File: pages/02.Setup/03.Cron Jobs/docs.en.md [hunk 2]
```diff
@@ -139,6 +139,25 @@ Use ‘--gdpr’ flag to delete data to fulfill GDPR European regulation. This w
...
+
+MaxMind requires to keep a "do not sell" list up to date and remove all data that were added by those IP addresses in the past from MaxMind.
```
Comment (RCheesley) [inline pages/02.Setup/03.Cron Jobs/docs.en.md:144]:
```suggestion
MaxMind requires users to keep a "Do Not Sell" list up to date, and remove all data that was added relating to those IP addresses in the past from MaxMind.
```

File: pages/02.Setup/03.Cron Jobs/docs.en.md [hunk 3]
```diff
@@ -139,6 +139,25 @@ Use ‘--gdpr’ flag to delete data to fulfill GDPR European regulation. This w
...
+
+See more details in the official MaxMind website: https://blog.maxmind.com/tag/ccpa/
```
Comment (RCheesley) [inline pages/02.Setup/03.Cron Jobs/docs.en.md:146]:
```suggestion
See more details in the official MaxMind website: [https://blog.maxmind.com/tag/ccpa/][maxmind-ccpa]
```

Comment (RCheesley) [reply pages/02.Setup/03.Cron Jobs/docs.en.md:146]: Please add the link macro at the end of the page:

`[maxmind-ccpa]: <https://blog.maxmind.com/tag/ccpa/>`

File: pages/02.Setup/03.Cron Jobs/docs.en.md [hunk 4]
```diff
@@ -139,6 +139,25 @@ Use ‘--gdpr’ flag to delete data to fulfill GDPR European regulation. This w
...
+
+It's recommended to run those 2 commands once/week. One after another.
```
Comment (RCheesley) [inline pages/02.Setup/03.Cron Jobs/docs.en.md:148]:
```suggestion
It is recommended to run these two commands once per week, one after another.
```

File: pages/02.Setup/03.Cron Jobs/docs.en.md [hunk 5]
```diff
@@ -139,6 +139,25 @@ Use ‘--gdpr’ flag to delete data to fulfill GDPR European regulation. This w
...
+
+
+```
+php /path/to/mautic/bin/console mautic:donotsell:download
+```
+This command will download the database of do not sell IP addresses from MaxMind.
```
Comment (RCheesley) [inline pages/02.Setup/03.Cron Jobs/docs.en.md:154]:
```suggestion
This command will download the database of Do Not Sell IP addresses from MaxMind.
```

File: pages/02.Setup/03.Cron Jobs/docs.en.md [hunk 6]
```diff
@@ -139,6 +139,25 @@ Use ‘--gdpr’ flag to delete data to fulfill GDPR European regulation. This w
...
+
+```
+php /path/to/mautic/bin/console mautic:max-mind:purge
+```
+This command will find data in the database that were loaded from MaxMind's do not sell IP addresses and deletes them.
```
Comment (RCheesley) [inline pages/02.Setup/03.Cron Jobs/docs.en.md:159]:
```suggestion
This command will find data in the database that was loaded from MaxMind's Do Not Sell IP addresses and deletes the data.
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
A couple of recommendations - sorry for the delay in getting this merged!

**Review by npracht — APPROVED** [review]
👍🏽

File: pages/02.Setup/03.Cron Jobs/docs.en.md [hunk 7]
```diff
@@ -139,6 +139,25 @@ Use ‘--gdpr’ flag to delete data to fulfill GDPR European regulation. This w
...
 ```
 
+### MaxMind CCPA compliance
+
+MaxMind requires users to keep a "Do Not Sell" list up to date, and remove all data that was added relating to those IP addresses in the past from MaxMind.
+
+See more details in the official MaxMind website: [https://blog.maxmind.com/tag/ccpa/][maxmind-ccpa]
+
+It is recommended to run these two commands once per week, one after another.
+
```
Comment (RCheesley) [inline pages/02.Setup/03.Cron Jobs/docs.en.md:149]:
```suggestion
```

**Review by RCheesley — APPROVED** [review]

**Pull request merged**